actions/cabal-ghc-compat/action.yml

```diff
@@ -0,0 +1,80 @@
+name: Check Cabal/GHC compatibility
+description: >
+  Check that the Cabal configuration file is compatible with the version of the
+  Cabal library bundled with GHC.
+inputs:
+  dirs:
+    description: Directories in which to run
```

Comment:
relative to?

Reply:
Noted that this is relative to the repo root.